src/SFML.Graphics/RenderTarget.cs

```diff
@@ -16,6 +17,14 @@ public interface RenderTarget
         ////////////////////////////////////////////////////////////
         Vector2u Size { get; }
 
+
+        ////////////////////////////////////////////////////////////
+        /// <summary>
+        /// Tell if the render target will use sRGB encoding when drawing on it
+        /// </summary>
+        ////////////////////////////////////////////////////////////
+        bool IsSrgb { get; }
```

Comment:
Where is this set?

Reply:
Ah my bad, I didn't realized this is on RenderTarget and that RenderTarget is an interface 😄